schema/deploy/warehouse/organism.sql

```diff
@@ -0,0 +1,36 @@
+-- Deploy seattleflu/schema:warehouse/organism to pg
+-- requires: hstore
+-- requires: ltree
+-- requires: warehouse/schema
+
+begin;
+
+set local search_path to warehouse, public;
+
+create table organism (
+    organism_id integer primary key generated by default as identity,
+    lineage ltree not null unique,
+    identifiers hstore,
```

Comment:
What's the advantage of using `hstore` over `jsonb` or `json`?

Reply:
It's more constrained. :-) I intentionally wanted the column to only support key-value pairs, not arbitrary data structures.